edmq_backend/src/scripts/generate_song_list.py

```diff
@@ -0,0 +1,34 @@
+import requests
+import json
+import shazam_data
+
+# Youtube API Request
+part = 'snippet'
+playlist_id = 'PLu-qQLnSm9zQwyCVh_c9uXJ6PDbEFBU8P'
+api_key = 'AIzaSyAJtLJ-uNEpBLZqWHngF_Xpji2NmHFsqSQ'
+playlist_url = 'https://youtube.googleapis.com/youtube/v3/playlistItems?part=' + part + '&playlistId=' + playlist_id + '&key=' + api_key
+
+# Load response into dictionary
+yt_response = requests.get(playlist_url).text
+playlist_dict = json.loads(yt_response)
```

Comment:
can you provide a comment describing what this json looks like

Reply:
Returns playlist details:

> - kind (item type)
> - etag (
> - items

items array contains details for each youtube video in playlist:

> - title 
> - description 
> - resourceids (where I pull videoid).
> - thumbnails

Reply:
i suggest extracting the json from the youtube response and then putting it in a comment here